crates/namada/src/ledger/protocol/mod.rs

```diff
@@ -178,51 +178,150 @@ impl From<Error> for DispatchError {
     }
 }
 
+/// Arguments for transactions' execution
+pub enum DispatchArgs<'a, CA: 'static + WasmCacheAccess + Sync> {
+    /// Protocoli tx data
```

Comment:
```suggestion
    /// Protocol tx data
```
(but I do like protocoli better tbh)